examples/bri-3/README.md

```diff
@@ -85,7 +86,7 @@ Run the following commands in the terminal:
 
 $ nats context add local --description "Localhost" # adds localhost server to nats cli
 
-$ nats pub general "<message>" # publishes new message on the general subject. Assumes two bpi subjects are added to the db with proper PKs.
+$ nats --user <username> --password <password> pub general "<message>" # publishes new message on the general subject. Assumes two bpi subjects are added to the db with proper PKs.
```

Comment:
should this docs be grouped with messaging section above?

Reply:
Fixed